### PR TITLE
Improve debian compile instructions

### PR DESCRIPTION
**Pull request by AndySchroder**
fix and clean up debian compilation instructions

also helps resolve https://github.com/estkme-group/lpac/issues/75

### CODE REVIEW

**Comment by AndySchroder** [comment]
Please close https://github.com/estkme-group/lpac/pull/110 first. It will be much easier to see how compact this change is.

**Comment by damonto** [comment]
You should checkout a new branch on the branch of main.

**Comment by AndySchroder** [comment]
> You should checkout a new branch on the branch of main.

done

File: docs/DEVELOPERS.md [hunk 1]
```diff
@@ -8,9 +8,31 @@ lpac is written with C99 and compatible with [SGP.22 version 2.2.2](https://www.
 
 #### Debian/Ubuntu
 
-With `build-essential` `cmake` `git` `g++` `libpcsclite-dev` `libcurl4-openssl-dev` installed.  
-Run `cmake` and `make`, and then all things will be in `output` directory.  
-If you want to get a Deb package, pass `-DCPACK_GENERATOR=DEB` when invoke `cmake`.
+- Install dependencies and download source code:
```
Comment (CoelacanthusHex) [inline docs/DEVELOPERS.md:11]: This is documents for **developers**, not normal users.

**Review by CoelacanthusHex — CHANGES_REQUESTED** [review]

**Comment by CoelacanthusHex** [comment]
This PR doesn't match our design of document structure.

File: docs/DEVELOPERS.md (hunk 1 above)
Comment (AndySchroder) [reply docs/DEVELOPERS.md:11]: this makes it easier for developers to understand.

**Pull request closed without merging**